### PR TITLE
change sample weight: calculated only once

### PR DESCRIPTION
**Pull request by dymczykm**

### CODE REVIEW

File: voxblox/include/voxblox/integrator/tsdf_integrator_fast.h [hunk 1]
```diff
@@ -77,6 +77,17 @@ class TsdfIntegrator {
     return 0.0;
   }
 
+  inline float getVoxelWeight(const Point& point_C) const {
```
Comment (schneith) [inline voxblox/include/voxblox/integrator/tsdf_integrator_fast.h:80]: rettype FloatingPoint

File: voxblox/include/voxblox/integrator/tsdf_integrator_fast.h [hunk 2]
```diff
@@ -109,9 +120,11 @@ class TsdfIntegrator {
     const float new_weight = tsdf_voxel->weight + updated_weight;
     tsdf_voxel->color = Color::blendTwoColors(
         tsdf_voxel->color, tsdf_voxel->weight, color, updated_weight);
-    const float new_sdf =
-        (sdf * updated_weight + tsdf_voxel->distance * tsdf_voxel->weight) /
-        new_weight;
+
+    const float sdf_times_weight = sdf * updated_weight;
```
Comment (schneith) [inline voxblox/include/voxblox/integrator/tsdf_integrator_fast.h:124]: FloatingPoint, and below

File: voxblox/include/voxblox/integrator/tsdf_integrator_fast.h (hunk 1 above)
Comment (mfehr) [reply voxblox/include/voxblox/integrator/tsdf_integrator_fast.h:80]: actually no, helen reminded me that the FloatingPoint typedef was originally  used to determine the precision of Points in space, not the signed distance function value

File: voxblox/include/voxblox/integrator/tsdf_integrator_fast.h (hunk 2 above)
Comment (mfehr) [reply voxblox/include/voxblox/integrator/tsdf_integrator_fast.h:124]: same

**Pull request closed without merging**